web/client/components/charts/__tests__/sample_data.js

```diff
@@ -0,0 +1,27 @@
+
+export const DATASET_1 = {
```

Comment:
in general do we have this in the resources folder, or not?

Reply:
I didn't want to put it so far from the specific usage, or we loose the reason why some data is in global test-resources folder. They was originally inline, I put them out only to separate data sets, that can grow, from effective tests.
I should put them out also if are common things (like generic responses of well know services). Do you think I should move them anyway?

Reply:
no it is ok, just wondering when to use test-resources and when not